Assets/Shopify/Unity/SDK/CartLineItems.cs

```diff
@@ -161,6 +161,14 @@ public class CartLineItem {
     /// Is used to add, update, or delete line items in a <see ref="Cart">Cart </see>.
     /// </summary>
     public class CartLineItems {
+        public enum LineItemChangeType {
+            add,
```

Comment:
Nit: Looks like we've either used CamelCasing or CAP_CASING for enums. Should standardize around CamelCasing?

Reply:
Sorry do you mean Pascal Case? I'm cool with Pascal.